Src/IronPython/Modules/Builtin.cs

```diff
@@ -1465,20 +1465,15 @@ public class mapobject : IEnumerable {
         }
 
         public static List sorted(CodeContext/*!*/ context, object iterable) {
-            return sorted(context, iterable, null, null, false);
+            return sorted(context, iterable, null, false);
         }
 
-        public static List sorted(CodeContext/*!*/ context, object iterable, object cmp) {
-            return sorted(context, iterable, cmp, null, false);
-        }
-
-        public static List sorted(CodeContext/*!*/ context, object iterable, object cmp, object key) {
-            return sorted(context, iterable, cmp, key, false);
+        public static List sorted(CodeContext/*!*/ context, object iterable, object key) {
```

Comment:
We could probably just get rid of this overload and the one about since we're just calling with the default values.